packages/plugin-lighthouse/src/lib/utils.ts

```diff
@@ -48,3 +49,22 @@ export function getLighthouseCliArguments(
 
   return objectToCliArgs(argsObj);
 }
+
+export class AuditsNotImplementedError extends Error {
+  constructor(auditSlugs: string[]) {
+    super(`audits: "${auditSlugs.join(', ')}" not implemented`);
+  }
+}
+
+export function validateOnlyAudits(
+  audits: Audit[],
+  onlyAudits: string | string[],
+): audits is Audit[] {
+  const missingAudtis = toArray(onlyAudits).filter(
+    slug => !audits.some(wS => wS.slug === slug),
```

Comment:
No clue what `wS` means.

```suggestion
    slug => !audits.some(audit => audit.slug === slug),
```